server/users/views.py

```diff
@@ -100,6 +100,9 @@ def recover_password(request: HttpRequest) -> JsonResponse:
     )
     recover_password.save()
 
+    userprofile = UserProfile.objects.get(user=user)
+    locale = userprofile.locale.code
+
```

Comment:
This code is repeated

Reply:
I have reverted the code to the original version (yours) by removing my changes.